src/presets/firebase/preset.ts

```diff
@@ -68,4 +77,37 @@ const firebase = defineNitroPreset(
   }
 );
 
-export default [firebase] as const;
+const firebaseAppHosting = defineNitroPreset(
+  {
+    extends: "node-server",
+    serveStatic: true,
+    hooks: {
+      async compiled(nitro) {
+        await writeFile(
+          join(nitro.options.rootDir, ".apphosting/bundle.yaml"),
+          stringifyYAML({
+            version: "v1",
+            runConfig: {
+              runCommand: "node .output/server/index.mjs",
```

Comment:
Note: We need to somehow tell app hosting to ***only** deploy this

Reply:
We will look into how we might implement only deploying .output